src/main/java/org/mozilla/iot/webthing/Utils.java

```diff
@@ -30,4 +24,32 @@ public static String timestamp() {
         String now = Instant.now().toString().split("\\.")[0];
         return now + "+00:00";
     }
+
+    /**
+     * Get all IP addresses.
+     *
+     * @return List of addresses.
+     */
+    public static List<String> getAddresses() {
+        Set<String> addresses = new HashSet<>();
+        try {
+            for (NetworkInterface iface : Collections.list(NetworkInterface.getNetworkInterfaces())) {
+                for (InetAddress address : Collections.list(iface.getInetAddresses())) {
+                    String s = address.getHostAddress().split("%")[0];
```

Comment:
A comment explaining how these lines work would be nice for the next person to look at it

Reply:
Done.